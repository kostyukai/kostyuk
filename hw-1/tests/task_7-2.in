1.12512525
