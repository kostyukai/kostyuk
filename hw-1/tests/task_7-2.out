00111111111111111001000000000100000110101010110001010011101100000101001110110000
